storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -29,7 +29,8 @@ include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
 
 [IMPORTANT]
 ====
-{product-title} defaults to using an in-tree, or non-CSI, driver to provision AWS EBS storage. This in-tree driver will be removed in a subsequent update of {product-title}. Volumes provisioned using the existing in-tree driver are planned for migration to the CSI driver at that time.
+{product-title} defaults to using an in-tree, or non-CSI, driver to provision AWS EBS storage. For {product-title}, automatic migration from AWS EBS in-tree to the CSI driver is available as a Technology Preview (TP) feature.
```

Comment:
* I think you can remove the "For {product-title}" as it's pretty redundant (you said it in the previous sentence, and the whole doc set is for OCP).
* We don't abbreviate it as (TP), so you can remove that.

Other instances of these if you can please check the rest of the PR.